test/types/enum/scope.chpl

```diff
@@ -0,0 +1,6 @@
+enumScopeTest(2);
+
+proc enumScopeTest(param x) {
```

Comment:
Looking at the original issue, you should probably add a test for the type argument case, too, in case someone accidentally forgets to fix both (since type and param arguments follow different paths)